Modules/OrgUnit/classes/Staff/class.ilOrgUnitStaffGUI.php

```diff
@@ -1,414 +0,0 @@
-<?php
-/**
- * This file is part of ILIAS, a powerful learning management system
- * published by ILIAS open source e-Learning e.V.
- *
- * ILIAS is licensed with the GPL-3.0,
- * see https://www.gnu.org/licenses/gpl-3.0.en.html
- * You should have received a copy of said license along with the
- * source code, too.
- *
- * If this is not the case or you just want to try ILIAS, you'll find
- * us at:
- * https://www.ilias.de
- * https://github.com/ILIAS-eLearning
- *
- ********************************************************************
- */
-/* Copyright (c) 1998-2013 ILIAS open source, Extended GPL, see docs/LICENSE */
-
-/**
- * Class ilOrgUnitStaffGUI
- * @author            Oskar Truffer <ot@studer-raimann.ch>
- * @author            Martin Studer <ms@studer-raimann.ch>
- * Date: 4/07/13
- * Time: 1:09 PM
- * @ilCtrl_Calls      ilOrgUnitStaffGUI: ilRepositorySearchGUI
- */
-class ilOrgUnitStaffGUI
```

Comment:
I don't think you can delete these. They seemed to be [called from `ilObjOrgUnitGUI`](https://github.com/ILIAS-eLearning/ILIAS/blob/release_8/Modules/OrgUnit/classes/class.ilObjOrgUnitGUI.php).

Reply:
I looked at the (removed) GUIs - they double ilOrgUnitUserAssignmentGUI and I'm quite sure they are not being called anymore